dev/docker/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.8"
 services:
   node:
-    image: xmtp/node-go:latest
+    image: xmtp/node-go@sha256:fba4c5f02203d90f25c91982fa1343d1d73c6bfbc3e3da16bd2ccbeef0a0f0f1
```

Comment:
Before we can merge this we will need to get the changes reverted in https://github.com/xmtp/xmtp-node-go/pull/447 back in to `main`